src/Host.cpp

```diff
@@ -169,6 +167,8 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 , mHaveMapperScript(false)
 , mAutoAmbigousWidthGlyphsSetting(true)
 , mWideAmbigousWidthGlyphs(false)
+, mSColonOnlySCRColCodesHaveColSpace(false)
```

Comment:
I *really* wanted something shorter and in this case I am open to suggestions!

Reply:
Would `mSGRCodeHasColSpaceId` be short enough for your tender sensibilities?

Reply:
:grin: how about `mSGRColonSeparator`?

Reply:
For what? This role - which means: when handling SGR 38 or 48 codes of type 2 (16M RGB colours) or 3 {16M CMY - not used but passed-over and discarded} or 4 {4.3B CMYK - also not used but passed over} ***has an extra parameter*** which is in the **third** position (but which is expected to an default {i.e. empty, no other characters}) and thus needs to be passed over to get to the *colour* components - **but ONLY when they are delimited only with semicolons `;`** (and not sub-divided with colons `:`)...

... so calling it `mSGRColonSeparator` is confusing because I (and more importantly I guess *others*) might expect *that* to be used somehow to flag that SGR sequences have been spotted which use the colon as a (sub-)separator.